core/src/components/back-button/back-button.tsx

```diff
@@ -53,6 +53,12 @@ export class BackButton implements ComponentInterface, ButtonInterface {
    */
   @Prop() type: 'submit' | 'reset' | 'button' = 'button';
 
+  /**
+   * When using a router, it specifies the transition animation when navigating to
+   * another page using `href`.
```

Comment:
Isn't this also used on click though? Not only when using an `href`? Since it can be undefined:

```
export const openURL = async (url: string | undefined | null, ev: Event | undefined | null, direction: RouterDirection): Promise<boolean> => {
```

Reply:
Oops yeah. I copied and pasted that comment from all the other props. For back button it should always apply.